package.json

```diff
@@ -388,20 +388,21 @@
     "eslint": "^5.5.0",
     "react": "^16.5.0",
     "react-dom": "^16.5.0",
-    "react-redux": "^5.0.7",
-    "redux": "^4.0.0",
     "sinon": "^7.0.0"
   },
   "dependencies": {
     "@folio/react-intl-safe-html": "^1.0.2",
+    "final-form": "^4.18.2",
+    "final-form-arrays": "^3.0.1",
     "lodash": "^4.17.4",
     "prop-types": "^15.5.10",
     "query-string": "^5.0.0",
+    "react-final-form": "^6.3.0",
+    "react-final-form-arrays": "^3.1.0",
     "react-hot-loader": "^4.3.12",
     "react-intl": "^2.3.0",
     "react-router": "^4.0.0",
-    "react-router-dom": "^4.0.0",
-    "redux-form": "^7.0.3"
+    "react-router-dom": "^4.0.0"
   },
   "peerDependencies": {
     "@folio/stripes": "^2.1.0",
```

Comment:
To make sure this include a `stripes` release with `stripes-final-form`, this should be bumped to 
```
    "@folio/stripes": "^2.9.0",
```

Reply:
Thanks @zburke. I just fixed it. I'm going to merge this in. Let's see if this actually works.